hazelcast/src/main/java/com/hazelcast/internal/plugin/multicast/MulticastProperties.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2008-2016, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.plugin.multicast;
+
+import com.hazelcast.config.properties.PropertyDefinition;
+import com.hazelcast.config.properties.PropertyTypeConverter;
+import com.hazelcast.config.properties.SimplePropertyDefinition;
+import com.hazelcast.config.properties.ValidationException;
+import com.hazelcast.config.properties.ValueValidator;
+
+import static com.hazelcast.config.properties.PropertyTypeConverter.STRING;
+
+/**
+ * Defines the name and default value for Multicast Plugin properties
+ */
+public final class MulticastProperties {
+
+
+    public static final PropertyDefinition PORT = property("port", STRING);
```

Comment:
@mdogan , yes you are right, this is a bug, i will send a PR with the tests. 
many thanks.